R/unreachable_code_linter.R

```diff
@@ -14,6 +15,18 @@
 #'   linters = unreachable_code_linter()
 #' )
 #'
+#' code_lines <- "f <- if (FALSE) {\n 2 + 2\n}"
+#' lint(
+#'   text = code_lines,
+#'   linters = unreachable_code_linter()
+#' )
+#'
+#' code_lines <- "f <- while (FALSE) {\n 2 + 2\n}"
+#' lint(
+#'   text = code_lines,
+#'   linters = unreachable_code_linter()
+#' )
+#'
 #' # okay
 #' code_lines <- "f <- function() {\n  return(1 + 1)\n}"
 #' writeLines(code_lines)
```

Comment:
Feel free to refactor these. They stem from a time before `text=` existed.

Reply:
that's not it -- the idea is these examples run by example(). it is hard to parse what the code actually looks like when there's escapes and newlines. for max readability, writeLines() will show you the exact code being linted.

Reply:
Here's a good example:

https://lintr.r-lib.org/reference/brace_linter.html

`writeLines()` helps the user grok the spacing & newlines without having to build a file mentally